tst/EnergyPlus/unit/EMSManager.unit.cc

```diff
@@ -976,310 +976,418 @@ TEST_F( EnergyPlusFixture, EMSManager_TestFuntionCall ) {
 			DataRuntimeLanguage::TrendVariable( i ).TrendValARR( 4 ) = 4.4;
 		}
 
-		EMSManager::ManageEMS( DataGlobals::emsCallFromHVACIterationLoop, anyRan );
+		EMSManager::ManageEMS( DataGlobals::emsCallFromHVACIterationLoop, anyRan ); // process trend functions again using above data
 		EXPECT_TRUE( anyRan );
 
 		int index( 0 );
 		int offset( 25 ); // first 24 values in ErlExpression() are key words + 1 EMS global variable
 		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 1 ).Operator, FuncRound );
 		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 1 ).NumOperands, 1 );
-//		EXPECT_EQ( size( DataRuntimeLanguage::ErlExpression( 1 ).Operand ), 1 ); // why doesn't this compile? ... Array1D< ErlValueType > Operand
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 1 ).Operand.size(), 1 );
```

Comment:
I think these lines with the `size()` check are what throws the build warning.  The `size()` call gives a const unsigned long, and the literal `1` is inferred as a constant integer.  You can see this demonstrated in the Objexx unit test code [here](https://github.com/NREL/EnergyPlus/blob/d5fe22242257821429bbb5123dd69bdc8dce5de1/third_party/ObjexxFCL/tst/unit/Array1.unit.cc#L35).  Just change `1` to `1u` and it will be created as an unsigned and the build warning should go away.